cmdstanpy/__init__.py

```diff
@@ -0,0 +1,19 @@
+import atexit
+import os.path
+import shutil
+import tempfile
+
+
+CMDSTAN_PATH = os.path.abspath(os.path.join('.', 'releases', 'cmdstan'))
+TMPDIR = tempfile.mkdtemp()
```

Comment:
for the record, I still find this weird but it's orthogonal to the overall PR and can be improved in the future.

Reply:
agreed.